plans/build.fmf

```diff
@@ -0,0 +1,29 @@
+provision:
+  hardware:
+    cpu:
+      processors: ">= 2"
+    memory: ">= 6 GB"
+    virtualization:
+      is-supported: true
+prepare:
+  - how: install
+    package:
+      - cargo
```

Comment:
Can we change this to use `dnf builddep bootc` for the base requirements?

Reply:
Sure. It's moved to `build-tmt.fmf`.